Formula/sip.rb

```diff
@@ -31,14 +31,23 @@ def install
     end
 
     Language::Python.each_python(build) do |python, version|
-      # Note the binary `sip` is the same for python 2.x and 3.x
+      # Avoid picking up /Application/Xcode.app paths
+      ENV.delete("SDKROOT")
       system python, "configure.py",
                      "--deployment-target=#{MacOS.version}",
                      "--destdir=#{lib}/python#{version}/site-packages",
                      "--bindir=#{bin}",
                      "--incdir=#{include}",
                      "--sipdir=#{HOMEBREW_PREFIX}/share/sip"
       system "make"
+      # Use opt_prefix instead of keg path with brewed pythons
+      if version == "2.7" && Formula["python"].installed?
+        inreplace "sipconfig.py", %r{#{HOMEBREW_PREFIX}/Cellar/python/[\d\._]+},
+          "#{HOMEBREW_PREFIX}/opt/python"
+      elsif version.to_f.to_i == 3
```

Comment:
I think we need the installed check here too since there are other python3s that can satisfy the requirement

Reply:
Right. Fixed.